decomp2d/cmake/gfortran_flags.cmake

```diff
@@ -0,0 +1,10 @@
+set(gfortran_flags
+  -cpp
+  -funroll-loops
+  -floop-optimize
+  -g
```

Comment:
The flag `-g` is used only for DEBUG builds ?

Reply:
yes